packages/manager/modules/netapp/src/dashboard/snapshot-policies/controller.js

```diff
@@ -0,0 +1,24 @@
+export default class OvhManagerNetAppSnapshotPoliciesCtrl {
+  $onInit() {
```

Comment:
**nitpick:** Maybe we can add a `TODO` comment here because it will be replaced by validated constant, isn't it?